### PR TITLE
Update differential gene expression database table using v10 results

### LINKED ISSUE #64: Update differential gene expression database table using v10 results
Current differential gene expression (DGE) database table was built using v9 DESeq results, which is incompatible with the TPM database table that was built using v10 TPM data.

The DGE database table needs to be updated using v10 DESeq results to be compatible with the v10 TPM database table, because the TPM boxplots on the sides of the DGE heatmaps are generated using the TPM database table.

This issue is blocked by <https://github.com/PediatricOpenTargets/ticket-tracker/issues/273>.

cc @taylordm @chinwallaa @afarrel

### PR DESCRIPTION
**Pull request by logstar**
# Pull Request Template

## Description

> The DGE database table needs to be updated using v10 DESeq results to be compatible with the v10 TPM database table, because the TPM boxplots on the sides of the DGE heatmaps are generated using the TPM database table.

- Resolved #64

Notes for reviewers:

- It is recommended to review this PR after <https://github.com/PediatricOpenTargets/OpenPedCan-api/pull/63> is merged to the main branch, because the file changes in this PR are based on the changes in <https://github.com/PediatricOpenTargets/OpenPedCan-api/pull/63>.
- In the review, please only add comments or suggestions that are directly related code correctness or documentation readability.
- This PR is not broken into smaller PRs, because all changes in this PR are required to test whether the API works as expected or not. Most of the file changes are test results and plots.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### Changed

- Updated database using v10 differential gene expression (DGE) data.
- Installed `libxml2` in `Dockerfile`.
- Updated tests for DGE heatmap endpoints, according to v10 DGE data.

## How Has This Been Tested?

- [x] Integration test

**Task link/Screenshot/Terminal returns**:

```text
$ ./tests/run_tests.sh
API base URL: http://localhost:8082
✔ |  OK F W S | Context
✔ | 453       | tests/r_test_scripts/test_endpoint_http.R [1003.3 s]

══ Results ══════════════════════
Duration: 1003.4 s
```

For more details about local testing environment, options, and resources, see `README.md` of this PR branch.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [NA] New and existing unit tests pass locally with my changes
- [NA] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [ ] I have emojis in my PR and commits 😉

cc @taylordm @chinwallaa

### CODE REVIEW

**Review by atlas4213 — APPROVED** [review]

**Comment by logstar** [comment]
Hi @atlas4213 . Thank you for reviewing and approving this PR.

I will merge this PR soon, after #63 is approved.

cc @chinwallaa

**Pull request merged**
